modules/openy_features/openy_node/modules/openy_node_social_post/config/install/field.field.node.social_post.field_sp_image.yml

```diff
@@ -0,0 +1,37 @@
+langcode: en
+status: true
+dependencies:
+  config:
+    - field.storage.node.field_sp_image
+    - node.type.social_post
+  module:
+    - image
+id: node.social_post.field_sp_image
+field_name: field_sp_image
+entity_type: node
+bundle: social_post
+label: Image
+description: ''
```

Comment:
add a description for housekeeper or grandma as a user, please

Reply:
done.